wordpress-mobile/WordPress-iOS
# MediaPicker: WPMediaCollectionViewCell placeholder design update

**NOTE:** This PR has a dependency on https://github.com/wordpress-mobile/MediaPicker-iOS/pull/247. (Right now we are pointing to a specific git commit in the pod file.)

This PR updates some UIAppearance API values for `WPMediaCollectionViewCell`  in `WordPressAppDelegate`. We also are bringing in new updates from the `WPMediaPicker` pod. The end result yields a new default placeholder design:

<img width="419" alt="screen shot 2017-09-29 at 12 28 57 pm" src="https://user-images.githubusercontent.com/154014/31029168-52e2446e-a516-11e7-86fb-194c204d5043.png">

Fixes https://github.com/wordpress-mobile/MediaPicker-iOS/issues/245

To test:

Run WPiOS on a variety of devices and sims. Verify everything looks ok in both the media picker(s) as well as the media library.

Needs review: @SergioEstevao 

/cc @iamthomasbishop
